src/orquestra/sdk/_base/cli/_dorq/_entry.py

```diff
@@ -154,6 +154,51 @@ def stop(wf_run_id: t.Optional[str], config: t.Optional[str]):
     action.on_cmd_call(wf_run_id, config)
 
 
+@workflow.command()
+@cloup.option("-c", "--config", type=str, multiple=True)
+@cloup.option(
+    "-a",
+    "--all",
+    is_flag=True,
+    flag_value=True,
+    help="Show all workflow runs that match the specified filters.",
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏I'm not sure what this help string means

Reply:
Updated in [e7fc2e5](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/14/commits/e7fc2e5008a07912ef7a116f96de3c53312ee6c4): "Assume that any unset filters are not required."

Reply:
I'm not sure I get why this option is needed, can you help me understand?
If I do `dorq wf list --state RUNNING` I would expect to list the running workflows, but from this help message, it looks like I would need to write `dorq wf list --state RUNNING --all` which looks confusing.

Reply:
There was a logic train that led here, but on second viewing everything we wanted to do is covered by `-h`. The all option is removed in [865560e](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/14/commits/865560e212714d574dd04dee1d22a407f25c2abd)